### ISSUE TITLE
Make an autoplay option

### ISSUE THREAD

**Issue by rozbb** [rozbb/readtomyshoe#10]
Make a selection box which, when selected, will autoplay the next article once the current one is done.

**Comment by rozbb** [comment]
To add on to this, it should be possible to reorder the queue. Unclear how to do this is in an accessible way.

**Comment by rozbb** [comment]
One option: have a "Reorder" toggle switch. When on, the buttons on the queue become "move up" and "move to top". Need to figure out how voiceover refocuses after a queue entry is moved.

**Comment by rozbb** [comment]
Lower priority now that there is a podcast interface.
